tests/test_interpolate.py

```diff
@@ -83,25 +84,25 @@ def test_align_torsion():
     We expect that decoupled terms have their force constants turned set to zero,
     while maintaining the same equilibrium angles for the *same* period.
     """
-    a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p = np.random.rand(16)
+    a, b, c, d, e, f, g, h, i, j, j, k, l, m, n = np.random.rand(15)
```

Comment:
Surprise: repeated random assignment to j

```suggestion
    a, b, c, d, e, f, g, h, i, j, k, l, m, n = np.random.rand(14)
```